### PR TITLE
R universe

### LINKED ISSUE #536: Why not set up r-universe?
It may be useful to have r-universe configured to install a development version of languageserver so that Windows and mac-os can do a binary install.

https://ropensci.org/blog/2021/06/22/setup-runiverse/
https://ropensci.org/blog/2021/03/04/r-universe-buildsystem/

### PR DESCRIPTION
**Pull request by renkun-ken**
Closes #536 

I set up a [repo](https://github.com/REditorSupport/universe) to contain the configuration of our [r-universe](https://reditorsupport.r-universe.dev) and updated README accordingly.

The repo that runs Github Actions to update and build the unvierse is <https://github.com/r-universe/reditorsupport>.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +40,10 @@ apk add --no-cache curl-dev g++ gcc libxml2-dev linux-headers make R R-dev
 install.packages("languageserver")
 ```
 
-The development version of `languageserver` could be installed by
+The latest development build of `languageserver` could be installed from our [r-universe](https://reditorsupport.r-universe.dev) repository:
 
 ```r
-# install.packages("remotes")
-remotes::install_github("REditorSupport/languageserver")
+install.packages("languageserver", repos="https://reditorsupport.r-universe.dev")
```
Comment (eitsupi) [inline README.md:46]: It should be `install.packages("languageserver", repos = c(reditorsupport = 'https://reditorsupport.r-universe.dev', getOption("repos")))` (to install dependent packages).

Comment (renkun-ken) [reply README.md]: Thanks!

**Comment by renkun-ken** [comment]
Should we also include `lintr` in the universe as we prefer?

File: README.md [hunk 2]
```diff
@@ -39,11 +40,13 @@ apk add --no-cache curl-dev g++ gcc libxml2-dev linux-headers make R R-dev
 install.packages("languageserver")
 ```
 
-The development version of `languageserver` could be installed by
+The latest development build of `languageserver` could be installed from our [r-universe](https://reditorsupport.r-universe.dev) repository:
 
 ```r
-# install.packages("remotes")
-remotes::install_github("REditorSupport/languageserver")
```
Comment (randy3k) [inline README.md:46]: `r-universe` only builds [daily](https://github.com/r-universe/reditorsupport/blob/master/.github/workflows/rebuilds.yml#L4), do we still want to include a session of installing the package from github source for users helping to test PRs?

Comment (renkun-ken) [reply README.md]: Yes. Fixed.

**Comment by renkun-ken** [comment]
I thought lintr could be in *our* r-universe. But a simpler way is just to specify multiple repos:

```r
install.packages("languageserver", repos = c(
    reditorsupport = "https://reditorsupport.r-universe.dev",
    rlib = "https://r-lib.r-universe.dev",
    getOption("repos")
))
```

**Pull request merged**